### ISSUE TITLE
Push v1 changes to NPM

### ISSUE THREAD

**Issue by hoodsy** [elgerlambert/redux-localstorage#32]
@elgerlambert 
Per the discussion in #13, it would be really nice if you could push the changes to npm.

Currently, I have to patch my version with the code from the PR.

**Comment by MattiSG** [comment]
+1

**Comment by stoikerty** [comment]
+1

**Comment by stoikerty** [comment]
@hoodsy @MattiSG 

Before it is merged into master, you can install v1 using `package@version`
```
npm install --save redux-localstorage@1.0.0-rc4
```

**Comment by hoodsy** [comment]
Interestingly enough, I was having the same error from the PR with rc4.
Copying in the PR elevated this error.

**Comment by TuckerCowie** [comment]
+1

**Comment by gabadi** [comment]
+1

**Comment by gingermusketeer** [comment]
@elgerlambert Anything I can do to help get the v1 release out?

**Comment by ezekielchentnik** [comment]
agreed, been patching ... anything i can do to help?

**Comment by seansfkelley** [comment]
This issue has been around for months, so I just went ahead and [published a fork](https://www.npmjs.com/package/redux-localstorage-fix-localstorage-fork) for the express purpose of getting this fix in. Hopefully I can obsolete it soon.

**Comment by elgerlambert** [comment]
Hi all,

A new version has finally been released, version 1.0.0-rc5 is now available on npm!

I'm really sorry I didn't do this sooner. For the past half year or so I wasn't working on any react/redux projects and became unsure about the direction 1.0 was heading. That's now changed and I've been using redux-localstorage in production for the past month, this has helped me feel the pain points and unblock myself.

Once again, my appologies. Thanks @seansfkelley for jumping in and publishing a fork! rc5 should resolve most of the issues and PR's that are open. Please try it out and let me know if you guys bump into any unresolved issues!

**Comment by seansfkelley** [comment]
Thanks! I added a deprecation notice and a link here to my fork and pushed that to npm.

**Comment by elgerlambert** [comment]
Thanks again @seansfkelley!
